src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -77,6 +82,19 @@ export class SmartSnippet extends Component {
     });
   };
 
+  /**
+   * The options for the SmartSnippet
+   * @componentOptions
+   */
+  static options: ISmartSnippetOptions = {
+    /**
+     * The maximum height a question answer can have in pixels.
+     * Any part of a question answering exceeding this height will be hidden.
+     * A "show more" button is added underneath an answer when some part of it is hidden.
+     */
+    maximumSnippetHeight: ComponentOptions.buildNumberOption({ defaultValue: 250, min: 0 })
```

Comment:
While it doesn't look great at 0 pixels, I don't see a use in disabling it. A minimum height of 0 means that the snippet is always going to be collapsed.